frontend/src/devicelist/ActionMenu.js

```diff
@@ -47,8 +47,8 @@ export default class ActionMenu extends React.Component {
                     keepMounted
                     open={Boolean(this.state.anchorElement)}
                     onClose={this.handleClose}
-                    anchorOrigin={{horizontal: 'right'}}
-                    transformOrigin={{ horizontal: 'right' }}
+                    anchorOrigin={{horizontal: 'right', vertical: 'center'}}
+                    transformOrigin={{ horizontal: 'right', vertical: 'center'}}
```

Comment:
was there a point to this change? on my end it doesn't display the popup below the action menu like in the mockups; is it used in the tests?

Reply:
I only added this to get rid of a warning that appears when `anchorOrigin` or `transformOrigin` do not have a `vertical` property. I may have put the wrong value though...

Reply:
Changed vertical to `top`, returns to the old behaviour in 374bc07